docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -732,6 +732,152 @@ Fault-tolerant execution support
 The connector supports :doc:`/admin/fault-tolerant-execution` of query
 processing. Read and write operations are both supported with any retry policy.
 
+
+Table functions
+---------------
+
+The connector provides the following table functions:
+
+table_changes
+^^^^^^^^^^^^^
+
+Allows reading Change Data Feed (CDF) entries to expose row-level changes
+between two versions of a Delta Lake table. When the ``change_data_feed_enabled``
+table property is set to ``true`` on a specific Delta Lake table,
+the connector records change events for all data changes on the table.
+In addition to columns present in the table the function returns the following
+values for each change event:
+
+* ``_change_type``
+    Gives the type of change that occurred. Possible values are ``insert``,
+    ``delete``, ``update_preimage`` and ``update_postimage``.
+
+* ``_commit_version``
+    Shows the table version for which the change occurred.
+
+* ``_commit_timestamp``
+    Represents the timestamp for the commit in which the specified change happened.
+
+.. code-block:: sql
```

Comment:
This shows how to run the table function.. and I think it should therefore be explained before preceding info that explains the returned data

We also need a entry sentence before the query